### ISSUE TITLE
What is your Machine and Runtime Configuration?

### ISSUE THREAD

**Issue by whatbeg** [gcr/torch-residual-networks#15]
What is your Machine (Runtime Env.) Configuration ? like number of GPUs, and some other configuration info. like true batchsize, etc.

And another question is, in part of "Alternate training strategies", you didn't add Adam and other optim. strategies, why? and how does these optim. stategies' initial learning rate are set ? Is it "Try one by one" ?

thks.
